client/js/components/day/day.js

```diff
@@ -34,11 +33,13 @@ var Day = React.createClass({
   render: function(){
     var gigs = this.state.gigs.map(function(gig, idx){
       return (
-              <Gig
-                information={gig.information}
-                staff={gig.staff}
-                key={idx} />
-              );
+        <div className={'card'}>
```

Comment:
you're right! do you want me to add a commit to this PR, or just make the change later?